lampod/src/ln/peer_manager.rs

```diff
@@ -132,12 +132,13 @@ impl LampoPeerManager {
             .clone()
             .ok_or(error::anyhow!("channel manager is None"))?;
         let alias = self.conf.alias.clone().unwrap_or_default();
-        let addr = self
-            .conf
-            .announce_addr
-            .clone()
-            .unwrap_or_else(|| "127.0.0.1".to_string());
-        std::thread::spawn(move || {
+        let addr = if let Some(addr) = self.conf.announce_addr.clone() {
+            addr
+        } else {
+            log::warn!("Address not specified inside conf, node is not reachable");
```

Comment:
```suggestion
```

Why a warning? if I want to stay private I do  not announce the address

Reply:
Should I put a `log::info!` or completely removed this line?

Reply:
There is no needed to add this message imho